lib/launcher.js

```diff
@@ -76,7 +76,8 @@ async function launch(options) {
   const params = [
     ['experiments', true],
     ['debugFrontend', options.debugFrontend],
-    ['sources.hide_add_folder', true]];
+    ['sources.hide_add_folder', true],
+    ['sources.hide_thread_sidebar', true]];
```

Comment:
Should we add logic that shows node process in this pull request?

Reply:
Done.